docs/book/advanced-guide/practical/stack-recipes.md

```diff
@@ -81,7 +81,105 @@ In addition to the underlying `terraform` functionality, these commands also off
 - ability to list all the available recipes conveniently before you choose to deploy any one of them.
 - checks to ensure that you have all the binaries/tools installed for running a recipe.
 - extensive logs and error messages that guide you in case any of the recipes fails or misbehaves.
-- option to automatically import a ZenML stack out of the components created after deploying a stack recipe.
+- option to automatically import a ZenML stack out of the components created
+  after deploying a stack recipe.
+
+To learn more about what you can do with the ZenML CLI, please refer to the [CLI
+docs](https://apidocs.zenml.io/latest/cli/).
+
+### Deploying stack components directly
```

Comment:
```suggestion
## Deploying stack components directly
```
I would make this a higher heading and have it precede the "Integration with ZenML CLI" block to make it more visible. the purpose of that block is mostly to provide an understanding of what additional checks and capabilities are added by using the CLI on top of normal terraform.

Reply:
Got it. Moved now.